### ISSUE TITLE
twitter-bootstrap UI problem

### ISSUE THREAD

**Issue by gundestrup** [litespeedtech/lscache-joomla#10]
When loading the website (and thereby the cache) on a desptop
It looks ok.
But when then loading the website on a cell phone UI, pictures are missing, and menu (collapserable) stop working.

It seems to me your caching solution is not device specific.

In joomla 3.8.5, to enable this behaviour (caching and device specific), you do the following
1) enable caching in global config
2) enable "Platform Specific Caching" in global config
3) install jotcache
4) enable "individual device caching" in jotcache (plugin)

You proberley need to adobt a device sensitive caching solution, since webite often have different looks dependen on what device it is loaded on....

**Comment by WuhuaChen** [comment]
Thanks for reporting.  We tested this issue exists.  to avoid this issue ,  it need to change .htaccess file, adding the following directives:
    RewriteCond %{HTTP_USER_AGENT} Mobile|Android|Silk/|Kindle|BlackBerry|Opera\ Mini|Opera\ Mobi 
    [NC] RewriteRule .* - [E=Cache-Control:vary=ismobile]

We updated your a2 testing site .htaccess file and found this issue solved.  We checked the calendar page ( https://a2.lungekursus.dk/kalender/maanedskalender/2018/3/-.html ) and found now it display different content in desktop and mobile devices .
